config/rbac/role.yaml

```diff
@@ -243,7 +251,13 @@ rules:
   - storageclasses
   verbs:
   - '*'
+  - create
+  - delete
   - get
+  - list
+  - patch
+  - update
```

Comment:
why this permission is required?

Reply:
For the `createOrReplaceStorageClass` function (will remove these permissions if the function logic changes).